src/sagemaker_training/environment.py

```diff
@@ -358,6 +358,27 @@ def num_cpus():  # type: () -> int
     return multiprocessing.cpu_count()
 
 
+def validate_smddp_smddpmprun():  # type: () -> bool
+    """Whether smddp and smddpmprun are installed.
+
+    Returns:
+        bool: True if both are installed
+    """
+    try:
+        import smdistributed.dataparallel.torch.torch_smddp
+
+        print(smdistributed.dataparallel.torch.torch_smddp)
```

Comment:
Is the print statement necessary here?

Reply:
It was added to avoid `unused import` complaint from `flake8`. Figured out a way to solve it. Will update it.